pypesto/ensemble/util.py

```diff
@@ -316,9 +316,7 @@ def read_ensemble_prediction_from_h5(
                 bounds[key] = np.array(bounds[key])
                 continue
             x_names = decode_array(f[f'{key}/{X_NAMES}'][()])
-            condition_ids = np.array(
-                decode_array(f[f'{key}/condition_ids'][()])
-            )
+            condition_ids = decode_array(f[f'{key}/condition_ids'][()])
```

Comment:
Does this do anything? Fine to leave as-is, just curious.

```suggestion
            condition_ids = decode_array(f[f'{key}/condition_ids'])
```

Reply:
Also elsewhere there are `[:]` that probably don't do anything? But fine to merge I think

Reply:
Does not work without. I think it does return array instead of h5py dataset.